### PR TITLE
[WIP] Consume intellisense from package

### PR DESCRIPTION
**Pull request by AdamYoblick**
Consume intellisense from package instead of from winforms/wpf packages

### CODE REVIEW

**Comment by AdamYoblick** [comment]
@NikolaMilosavljevic This is my working PR for this work. Not done yet by any means, but I just wanted to show you how I was approaching it for our meeting tomorrow.

File: pkg/Directory.Build.targets [hunk 1]
```diff
@@ -262,6 +262,11 @@
     </ItemGroup>
   </Target>
 
+  <Target Name=IncludeIntellisenseFiles
```
Comment (RussKie) [inline pkg/Directory.Build.targets:265]:
```suggestion
  <Target Name="IncludeIntellisenseFiles"
```

**Comment by AdamYoblick** [comment]
I spoke with Nikola offline and he mentioned he heard this morning that for 6.0 the goal is to build intellisense directly from code, so this work might not be needed. I'm waiting to hear back from him.

**Comment by AdamYoblick** [comment]
Going to close this out since xml comments in code are going to be the source of truth in the future.

**Pull request closed without merging**